src/xtgeo/__init__.py

```diff
@@ -1,5 +1,4 @@
 # flake8: noqa
-# pylint: skip-file
```

Comment:
It seems ruff and pylint are not one-to-one: https://docs.astral.sh/ruff/faq/#how-does-ruffs-linter-compare-to-pylint should we be removing it?

I lean toward yes, as we aren't currently using it and who knows how out of date some of the pylint ignores could be. But @jcrivenaes should weight in.

Reply:
Ive not documented it the description, but ive removed all other linters.

https://github.com/equinor/xtgeo/pull/1062/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L81

Reply:
Tracking that, I am just questioning if it should be removed or not since it checks things ruff does not (and vice versa)